src/TableFunctions/TableFunctionObjectStorageCluster.h

```diff
@@ -33,6 +33,42 @@ struct HDFSClusterDefinition
     static constexpr auto storage_type_name = "HDFSCluster";
 };
 
+struct IcebergClusterDefinition
+{
+    static constexpr auto name = "icebergCluster";
+    static constexpr auto storage_type_name = "IcebergS3Cluster";
```

Comment:
```suggestion
    static constexpr auto storage_type_name = "IcebergCluster";
```

Reply:
Did the same as here https://github.com/ClickHouse/ClickHouse/blob/master/src/TableFunctions/TableFunctionObjectStorage.h#L68, do you think it's worth changing?

Reply:
On second thought, do we need `IcebergCluster` function? 
I believe we have iceberg & icebergS3 as we originally just had iceberg which was for S3 then extended it for other object storages. (iceberg is alias of icebergs3) 
what do you think?

Reply:
Seems so, let's just keep IcebergS3Cluster.